sklearn/svm/base.py

```diff
@@ -481,6 +497,21 @@ def _validate_targets(self, y):
 
         return np.asarray(y, dtype=np.float64, order='C')
 
+    def decision_function(self, X):
+        """Distance of the samples X to the separating hyperplane.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
```

Comment:
X : array-like, shape (n_samples, n_features)

can you go over the docstrings to make sure we use this convention here too?